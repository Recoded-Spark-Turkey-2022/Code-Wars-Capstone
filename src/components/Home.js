import React from "react";

const Home = () =>{
    return <h1>Naji</h1>
}

export default Home;