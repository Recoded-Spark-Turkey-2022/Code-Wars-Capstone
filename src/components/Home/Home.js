import Carousel from 'react-multi-carousel';
import BookAppointment from './Book an Appointment/bookAppointment';
import Therapist from './therapist/Therapist';
import CommuncicationCard from './communication/CommuncicationCard';
import TicketPage from './Purchasetickets/Purchasetickets';
import BlogSlider from '../blogs/SliderBlog';
import 'react-multi-carousel/lib/styles.css';

export default function Home() {
  return (
    <>
      <BookAppointment />
      <Therapist />
      <CommuncicationCard />
      <BlogSlider Carousel={Carousel} />
      <TicketPage />
    </>
  );
}
