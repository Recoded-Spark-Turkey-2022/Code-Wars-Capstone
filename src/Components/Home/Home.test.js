
import React from "react";
import renderer from "react-test-renderer";
import Home from "./Home";
import { MemoryRouter } from "react-router-dom";

it("renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <Home/>
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});