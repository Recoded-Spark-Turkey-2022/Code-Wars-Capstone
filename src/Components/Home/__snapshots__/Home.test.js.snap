// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
Array [
  <div
    className="relative h-screen "
    style={
      Object {
        "fontFamily": "Poppins, sans-serif",
      }
    }
  >
    <p>
       
    </p>
    <img
      alt="BackGround"
      className="absolute top-0 left-0 z-0 bg-cover   "
      src="BackGround.png"
    />
    <div
      className="z-10 w-6/12 translate-y-1/3  translate-x-2/3 "
    >
      <img
        alt="iltration"
        className="imgsofa"
        src="Towperson.svg"
      />
    </div>
    <div
      className=" absolute top-28  left-40"
    >
      <p
        className="paragraph"
      >
        We Are Here To
      </p>
      <p
        className="paragrapHelp"
      >
        Help
      </p>
      <a
        className="text-xl md:text-lg  lg:px-6 rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white "
        href="#Loge In"
      >
        BOOK AN APPOINTMENT
      </a>
    </div>
  </div>,
  <div
    className="h-screen"
  >
    <div
      className="bg-amber-200 h-screen "
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <h1
        className=" lg:text-5xl text-lg font-bold lg:w-10/12 sm:p-8  sm:pl-40  "
      >
        PROFESSIONAL, LICENSED, AND VETTED THERAPISTS THAT YOU CAN TRUST
      </h1>
      <img
        alt="therapists"
        className=" pb-8 pl-40"
        src="Therapistimg.svg"
      />
      <p
        className="pl-40 lg:text-2xl font-seri text-sm lg:w-11/12 lg:pb-5 sm:pb-16"
      >
        Tap into the worlds largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.
      </p>
      <button
        className="text-xl md:text-lg ml-40 lg:px-6 rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white "
        type="button"
      >
        BOOK AN APPOINTMENT
      </button>
    </div>
  </div>,
  <p
    className="text-4xl my-20 ml-40  "
    style={
      Object {
        "fontFamily": "Poppins, sans-serif",
      }
    }
  >
    WE CAN COMMUNICATE THROUGH
  </p>,
  <div
    className="flex gap-8  mb-20 mx-40    "
  >
    <div
      className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <img
        alt="cardsphoto"
        className="h-20 w-15 rounded-full mx-auto mb-4"
        src="phonecall.svg"
      />
      <h2
        className="text-2xl font-bold text-center mb-2"
      >
        Voice Call
      </h2>
      <p
        className="text-gray-700 text-center"
      >
        Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
      </p>
    </div>
    <div
      className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <img
        alt="cardsphoto"
        className="h-20 w-15 rounded-full mx-auto mb-4"
        src="chat.svg"
      />
      <h2
        className="text-2xl font-bold text-center mb-2"
      >
        Chat
      </h2>
      <p
        className="text-gray-700 text-center"
      >
        Need to talk to someone? Come have a chat with your therapist!
      </p>
    </div>
    <div
      className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <img
        alt="cardsphoto"
        className="h-20 w-15 rounded-full mx-auto mb-4"
        src="video.svg"
      />
      <h2
        className="text-2xl font-bold text-center mb-2"
      >
        Video Call
      </h2>
      <p
        className="text-gray-700 text-center"
      >
        For better experience therapists recommend video calls, but always remember that its a choice!
      </p>
    </div>
  </div>,
  <div
    className="bg-cyan-50 "
  >
    <p
      className="lg:ml-40 md:ml-20 py-20 lg:text-5xl md:text-2xl	sm:text-lg "
    >
      RECENT BLOGS
    </p>
    <div
      className="react-multi-carousel-list carousel-container  lg:ml-40 md:ml-20 w-[65rem] pb-40"
      dir="ltr"
    >
      <ul
        className="react-multi-carousel-track "
        onMouseDown={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseMove={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "overflow": "unset",
            "transform": "translate3d(0px,0,0)",
            "transition": "none",
          }
        }
      />
    </div>
  </div>,
  <div
    style={
      Object {
        "fontFamily": "Poppins, sans-serif",
      }
    }
  >
    <p
      className="lg:text-5xl md:text-2xl	sm:text-lg mt-10  lg:ml-40 md:ml-20"
    >
        Purchase tickets
    </p>
    <p
      className="lg:tex-xl md:text-lg sm:text-sm lg:ml-40 mb-6 md:ml-20"
    >
        PURCHASE TICKETS THAT CAN BE USED TO BOOK APPOINTMENTS!
    </p>
    <div
      className=" flex lg:gap-14  md:gap-8 sm:gap-4  flex-wrap  sm:felx-warp lg:ml-40 md:ml-20  lg:mb-20 md:mb-10 "
    >
      <div
        className="flex flex-col shadow-lg   px-10 py-16  text-center rounded-lg w-3/12  "
      >
        <h1
          className=" lg:text-5xl md:text-2xl	sm:text-sm  lg:pt-8 md:pt:6 sm:pt:4 "
        >
           
          5 TICKETS
        </h1>
        <p
          className="lg:text-2xl md:text-1xl	sm:text-sm text-stone-700 lg:pt-8 md:pt:6 sm:pt:4   lg:pb-6 md:pb:4 sm:pb:2 "
        >
           
          10$
        </p>
        <button
          className="lg:text-2xl md:text-1xl sm:text-sm rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white"
          onClick={[Function]}
          type="submit"
        >
           Purchase 
        </button>
      </div>
      <div
        className="flex flex-col shadow-lg   px-10 py-16  text-center rounded-lg w-3/12  "
      >
        <h1
          className=" lg:text-5xl md:text-2xl	sm:text-sm  lg:pt-8 md:pt:6 sm:pt:4 "
        >
           
          25 TICKETS
        </h1>
        <p
          className="lg:text-2xl md:text-1xl	sm:text-sm text-stone-700 lg:pt-8 md:pt:6 sm:pt:4   lg:pb-6 md:pb:4 sm:pb:2 "
        >
           
          40$
        </p>
        <button
          className="lg:text-2xl md:text-1xl sm:text-sm rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white"
          onClick={[Function]}
          type="submit"
        >
           Purchase 
        </button>
      </div>
      <div
        className="flex flex-col shadow-lg   px-10 py-16  text-center rounded-lg w-3/12  "
      >
        <h1
          className=" lg:text-5xl md:text-2xl	sm:text-sm  lg:pt-8 md:pt:6 sm:pt:4 "
        >
           
          50 TICKETS
        </h1>
        <p
          className="lg:text-2xl md:text-1xl	sm:text-sm text-stone-700 lg:pt-8 md:pt:6 sm:pt:4   lg:pb-6 md:pb:4 sm:pb:2 "
        >
           
          70$
        </p>
        <button
          className="lg:text-2xl md:text-1xl sm:text-sm rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white"
          onClick={[Function]}
          type="submit"
        >
           Purchase 
        </button>
      </div>
    </div>
    <div>
      <div
        className="flex flex-col shadow-lg  p-10  text-center rounded-lg w-9/12 lg:ml-40 md:ml-20 mb-20  "
      >
        <h1
          className=" lg:text-5xl md:text-2xl	sm:text-sm  lg:pt-8 md:pt:6 sm:pt:4 "
        >
          Are you a counselor? 
        </h1>
        <p
          className="lg:text-1xl md:text-xl	sm:text-sm text-stone-700 lg:pt-8 md:pt:6 sm:pt:4   lg:pb-6 md:pb:4 sm:pb:2 "
        >
          Interested in joining our mental health platform? You decide your schedule and how much you want to work, weâ€™ll take care of the client referrals and billing details!
        </p>
        <button
          className="lg:text-2xl md:text-1xl sm:text-sm rounded-md box-border py-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white w-2/6 m-auto "
          onClick={[Function]}
          type="submit"
        >
           Purchase 
        </button>
      </div>
    </div>
  </div>,
]
`;
